server/utils/APIWrapper.ts

```diff
@@ -21,7 +21,9 @@ interface Route<T> {
   ) => Promise<T>;
 }
 
-function APIWrapper<T>(routeHandlers: Partial<Record<HttpMethod, Route<T>>>) {
```

Comment:
There's no reason for each route to be bound to the same type T. This was creating issues with `npm run build` when two HTTP verbs from the same endpoint returned different types--for some reason, TypeScript did not derive a valid value for T in this instance.

Thus, Route<T> -> `Route<any>` or `Route<unknown>`. eslint doesn't like any, so I opted for `unknown`. We'll never use the the type returned by a route, since the values returned by these routes are serialized to JSON

Reply:
Ignore my last comment in this thread.  I just realized that even though what I said resolved the error it does not create a useful type.